types/route-parser/index.d.ts

```diff
@@ -2,8 +2,9 @@
 // Project: http://github.com/rcs/route-parser
 // Definitions by: Ian Ker-Seymer <https://github.com/ianks>, Bob Buehler <https://github.com/bobbuehler>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
 
-declare class Route {
+declare class Route<TMatch extends {} = { [i: string]: any }> {
```

Comment:
Please rename to `TParams`.
`reverse` takes a `params` argument and match returns `matched parameters`, so this is the type of the parameters and not the type of a match.

Reply:
Thanks for pointing out. I agree. This has now been updated.